### PR TITLE
Add more information about types to the plugins api

### PR DESCRIPTION
**Pull request by mineo**

### CODE REVIEW

File: website/frontend/templates/docs/plugin-api.html [hunk 1]
```diff
@@ -66,7 +76,10 @@ <h1 id="tagger_script_functions">Tagger Script Functions</h1>
       <p>
         To define new
         <a href="/docs/scripting/" title="Picard Qt/Scripting">tagger script</a> function use
```
Comment (mwiencek) [inline website/frontend/templates/docs/plugin-api.html]: s/function/functions/, if you don't mind fixing that too. (or s/new/a new/ above.)

**Comment by mwiencek** [comment]
:shipit:

File: website/frontend/templates/docs/plugin-api.html (hunk 1 above)
Comment (mineo) [inline website/frontend/templates/docs/plugin-api.html]: I've added the `s`.

**Comment by mwiencek** [comment]
I can't remember if you have access to sakura to update the website (if not you should send me your ssh key :)). It's updated with this now.

**Pull request merged**